### ISSUE TITLE
Cached content displays HTML tags

### ISSUE THREAD

**Issue by muesli** [RadhiFadlillah/shiori#57]
Instead of rendering the HTML I get to see text like this:

```
<p></p><div><div><p name="154c">Burton Howard Bloom, the eponymous hero of our tale, and, it has to be said, possessed of a name with a very pleasing cadence — <em>Bur-ton-Ho-ward-Bloom</em> 
...
```

**Comment by RadhiFadlillah** [comment]
Hi @muesli, thanks for the report.

This problem happened because I pushed commit [b16c5c](https://github.com/RadhiFadlillah/shiori/commit/b16c5cc363e5f06abaedfbf996fffcebddbe2788) to fix database problem on Go 1.8. 

It should be fixed by [e67397](https://github.com/RadhiFadlillah/shiori/commit/e6739728313c18091e2485bd127d36db9a00dc12).

**Comment by muesli** [comment]
Confirmed!
